apps/src/p5lab/spritelab/commands/behaviorCommands.js

```diff
@@ -14,10 +14,18 @@ export const commands = {
   draggableFunc(p5Inst) {
```

Comment:
this behavior has enough branching logic now that it should be covered with unit tests